src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.ts

```diff
@@ -322,14 +322,14 @@ export class RetrospectTaskModalComponent implements OnDestroy, AfterViewChecked
             {
                 headerName: 'Sprint Points',
                 field: 'SprintPoints',
-                editable: (params) => {
+                editable: (params: IsColumnFuncParams) => {
                     return isSprintEditable && params.data.Editable;
```

Comment:
@chetanyakan Please make sure you properly test the changes you make. Changing the flag name from `Editable` to `Current` is not done in all the places.